### PR TITLE
Add Deploying section and Puppet recipe #742

### PR DESCRIPTION
**Pull request by blues-man**
We would add a specific section for Deploying Crossbar with most
used configuration management tools as Puppet, Ansible, Salt etc
filling it with a Puppet recipe

### CODE REVIEW

File: docs/pages/installation/Deploying-Crossbar.io-with-Puppet.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+For an overview of all feature supported and more technical details, see the [GitHub repository](https://github.com/blues-man/crossbar-puppet).
+
+
+## Usage
+
+Installation:
+
+```console
+puppet module install bluesman-crossbar
+```
```
Comment (oberstet) [inline docs/pages/installation/Deploying-Crossbar.io-with-Puppet.md]: No, we need this under our control

**Comment by oberstet** [comment]
Thanks! I will have to try that first myself. Also, it is probably best to wait a little, since we are gonna restructure the installation docs and make it simple. That means, the recommended way of installing CB will be more prominent - this is via

1. first, our own binary packages
2. then, Docker
3. then, anything else is "advanced"

File: docs/pages/installation/Deploying-Crossbar.io-with-Puppet.md (hunk 1 above)
Comment (blues-man) [inline docs/pages/installation/Deploying-Crossbar.io-with-Puppet.md]: ah ok, fine

**Comment by blues-man** [comment]
Ok sounds good, in the while I just update the module that installs crossbar supporting both CentOS 7.x and Ubuntu 14.04  official packages and repo :)

**Comment by oberstet** [comment]
I had a quick look into your repo. It seems slightly more than trivial;) And since I don't use Puppet, and don't speak Ruby, this is a problem for me .. because what we say in the official docs MUST be proven, supported etc - we can't put stuff there we don't understand and hence cannot support.

Now, how about this: we add a page on the docs here for "community recipes" .. with a section for Puppet, which basically says "yeah, possible" .. please see here for a community repo that has it. And link to your repo.

**Comment by oberstet** [comment]
@blues-man this https://github.com/blues-man/crossbar-puppet/blob/master/manifests/repo.pp#L15 is highly appreciated. It gives users confidence that they are using exactly the bits we publish .. thanks for that!

**Comment by blues-man** [comment]
Eheh yes always better to query directly the key server :)

I didn't know where to exactly put Puppet recipes, but for this case and others, community recipes looks pretty fine!

**Comment by oberstet** [comment]
@goeddea pls see above .. some "community recipe page" or similar .. where we can point at @blues-man 's repo for CB deployment under Puppet.

**Pull request closed without merging**